src/main/webapp/script.js

```diff
@@ -29,21 +29,6 @@ async function getYouTubeComments() {
   return comments;
 }
 
-/*
- * Extracts video id from full url
- */ 
-function cleanseUrl(url) {
-  // Split web address from parameters, extract first parameter
-  // TODO: Add checks to make this work if video is not first parameter.
-  var videoId = url.split("?");
-  videoId = (videoId.length > 1) ? videoId[1].split("&")[0] : videoId[0];
-
-  // If param name present, remove it to isolate video Id.
-  videoId = videoId.replace("v=", "");
-  
-  return videoId;
-}
-
```

Comment:
It looks like you're removing the CleanseURL function from script.js and moving it into video_context.js for usage in  getVideoContext. This in and of itself is fine, however, won't removing this function from script.js cause an issue with the getComments, which also utilizes CleanseURL?